docs/getting-started-guides/network-policy.md

```diff
@@ -0,0 +1,131 @@
+---
+assignees:
+- caseydavenport
+
+---
+
+Kubernetes can be used to declare network policies which govern how Pods can communicate with each other.  This document helps you get started using the Kubernetes [NetworkPolicy API](/docs/user-guide/networkpolicies), and provides a demonstration thereof. 
+
+In this article we assume that that a Kubernetes cluster has been created with network policy support.  It's easy to create such a cluster using the [GCE guide](/docs/getting-started-guides/gce), setting the `NETWORK_POLICY_PROVIDER=calico` environment variable.
+
+e.g
+
+```shell
+export NETWORK_POLICY_PROVIDER=calico
+curl -sS https://get.k8s.io | bash
```

Comment:
As someone working on next-generation provisioning/bootstrap tool, you are already linking to GCE guide above, does this have to be here really? We will have to figure out how to propagate reference to new bootstrap tools through the docs, so may be it's okay to leave this as is for this PR, but thought I'd make sure we all start thinking about it.